stdlib/public/Concurrency/TaskCancellation.swift

```diff
@@ -0,0 +1,81 @@
+////===----------------------------------------------------------------------===//
+////
+//// This source file is part of the Swift.org open source project
+////
+//// Copyright (c) 2020 Apple Inc. and the Swift project authors
+//// Licensed under Apache License v2.0 with Runtime Library Exception
+////
+//// See https://swift.org/LICENSE.txt for license information
+//// See https://swift.org/CONTRIBUTORS.txt for the list of Swift project authors
+////
+////===----------------------------------------------------------------------===//
+
+import Swift
+import Dispatch
+@_implementationOnly import _SwiftConcurrencyShims
+
+// ==== Task Cancellation ------------------------------------------------------
+
+extension Task {
+  public struct CancellationError: Error {
+    // no extra information
+  }
+
+  /// Returns `true` if the task is cancelled.
+  public static func isCancelled() async -> Bool {
```

Comment:
U.S. English spelling is “canceled.”

Reply:
Good catch, thanks -- we'll need to adjust other writeups as well then.

For reference: It seems that `Cancellation` though in either spelling preference remains with double `L`

Reply:
Did some searching to sanity check before changing...

So it seems there is some prior art which is confusing, as NSOperation uses `cancelled` but also `cancel`. We could clean up the spellings here in the new API and stick to `canceled`. 

The current spelling of `cancelled` is also present in other sources (`Task.h` and other internal files) committed so far so whatever we decide we should apply consistently.

The current spelling is consistent with other uses in Concurrency, so let's decide and apply everywhere when -- perhaps a separate spelling cleanup PR?

Reply:
This is a common issue in terms of consistency. IMO very reasonable to deal with in a separate PR for consistency throughout all new APIs.

Reply:
I noted this down for myself to revisit this "everywhere" once we have a decision on the spellings -- closing thread for now.